### ISSUE TITLE
New Clojure wrapper for Awaitility

### ISSUE THREAD

**Issue by jordanrobinson** [awaitility/awaitility#272]
Hi there,

First off, thanks for all your hard work on Awaitility, we use it quite a bit where I work. Since almost all of our code is Clojure based, we created a small wrapper for Awaitility in Clojure, which we have just got round to open sourcing. It is located [on github here](https://github.com/jordanrobinson/awaitility-clj) and [on clojars here.](https://clojars.org/uk.co.jordanrobinson/awaitility-clj/versions/0.0.5)

I was wondering if you'd accept a PR for [this page](https://github.com/awaitility/awaitility/wiki/Getting_started) with some Clojure instructions pointing at the new wrapper? No worries if not and hope you're having a great day.

Also - I was going to send this over via Twitter as per your contact details but your DMs are closed I believe, so sorry if this isn't the right place for this.

Thanks,
Jordan

**Comment by johanhaleby** [comment]
That's really cool! I actually made a very simple version of awaitility myself in Clojure back in the day :) 

Would it be possible to even merge this into the main project? Or maybe it's easier for you to keep it separate and not involve maven etc? :)

**Comment by jordanrobinson** [comment]
Yeah I think the tricky think would be around maven - the key thing would be to have it available on clojars but if that would be possible I'd love to find a way to merge it into the main project.
